pull request to cerbero official site

balazskreith opened the pull request.

stefanalund commented:
Correct me if I'm wrong but this is only the prerequisites for building mprtp as part of OpenWebRTC, right? Have you also integrated mprtp in OpenWebRTC? Anyway, thanks, this could be pretty interesting.

balazskreith commented:
Hi,

Yes, cerbero contains the prerequisites (basically I added one additional
recipe and one dep into openwebrtc recipe).
And yes, I integrated it into OpenWebRTC as I changed the transport_agent (
https://github.com/multipath-rtp/openwebrtc).
The mprtp plugin development is ongoing right now, I am testing and
bugfixing.

Balázs

On 22 October 2015 at 09:33, Stefan Ålund notifications@github.com wrote:

> Correct me if I'm wrong but this is only the prerequisites for building
> mprtp as part of OpenWebRTC, right? Have you also integrated mprtp in
> OpenWebRTC? Anyway, thanks, this could be pretty interesting.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/EricssonResearch/cerbero/pull/33#issuecomment-150122367
> .

superdump commented:
This will be interesting to try out when it's ready. :) Thanks for making it visible for others!

For the sake of clarity in the project, I'm going to add a 'don't merge' label to this PR just so that we don't merge it by accident as it points to your fork. It is in no way a negative comment about the contribution other than that it's not yet ready to merge. :)
